Tests/JWSRSATests.swift

```diff
@@ -104,4 +133,26 @@ class JWSRSATests: RSACryptoTestCase {
         XCTAssertEqual(jws.signature.data(), signature)
     }
 
+    @available(*, deprecated)
+    private func performTestRSASerializationValidationAndDeserialization(algorithm: SignatureAlgorithm) {
```

Comment:
Ideally we can merge this without needing this new helper function. See above. Let me know what you think!